vhdl_ls/src/stdio_server.rs

```
@@ -64,24 +78,39 @@ pub fn start() {
         let stdin = io::stdin();
         loop {
             let request = read_request(&mut stdin.lock());
-            request_sender.send(request).unwrap();
+            match request_sender.send(request) {
+                Ok(_) => continue,
+                Err(_) => break,
+            }
         }
     });
 
-    // Spawn thread to write notificaitons to stdout
+    // Spawn thread to write notifications to stdout
     spawn(move || {
         let mut stdout = io::stdout();
         loop {
-            let response: String = response_receiver.recv().unwrap();
-            send_response(&mut stdout, &response);
+            match response_receiver.recv() {
+                Ok(response) => {
+                    send_response(&mut stdout, &response);
+                }
+                Err(_) => {
```

Maybe add an eprintln! of the error to not silently swallow it